sim_3d_volumes.py

```diff
@@ -0,0 +1,103 @@
+""" Generate 3D map of molecules
+"""
```

Comment:
To follow the style guideline closely this should end with a period and be only one line.